src/github.com/matrix-org/dendrite/clientapi/routing/login.go

```diff
@@ -131,3 +132,22 @@ func Login(
 		JSON: jsonerror.NotFound("Bad method"),
 	}
 }
+
+// check if device exists else create one
+func getDevice(
+	ctx context.Context,
+	r passwordRequest,
+	deviceDB *devices.Database,
+	acc *authtypes.Account,
+	localpart, token string) (dev *authtypes.Device, err error) {
```

Comment:
Please place the `) (dev ...` on it's on line to be consistent with other multi-line function declarations.

```
func getDevice(
  ctx context.Context,
  r passwordRequest,
  deviceDB *devices.Database,
  acc *authtypes.Account,
  localpart, token string
) (dev *authtypes.Device, err error) {
...
```